resources/media-controls.js

```diff
@@ -50,6 +37,22 @@
     }
   };
 
+  function generateMarkup(isAudioOnly) {
+    return `
+      <div class="controls">
+        <button id="play-pause-button"></button>
+        <input id="progress" type="range" value="0" min="0" max="100" step="1"></input>
+        <span id="position-duration-box" class="hidden">
+          <span id="position-text">#1</span>
+          <span id="duration"> / #2</span>
+        </span>
+        <button id="volume-switch"></button>
+        <input id="volume-level" type="range" value="100" min="0" max="100" step="1"></input>
+        ${isAudioOnly ? "" : '<button id="fullscreen-switch" class="fullscreen"></button>'}
```

Comment:
It's really not a big deal, but it would be very nice if we could change the icon depending on whether we are out (icon with arrows pointing out) or in (icon with arrows pointing in) of fullscreen mode. Or use a neutral icon instead.